src/services/api.service.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export const fetchUserData = async () => {
+  try {
+    const { data } = await axios.get(
+      `https://random-data-api.com/api/v2/users`
```

Comment:
nice job @divinirakiza  but you can store the API URL in a .env file to keep it separate from the code and make it easier to change in the future.